site/docs/v1/tech/lambdas/openid-connect-response-reconcile.adoc

```diff
@@ -68,6 +68,23 @@ During development if you want to get a better idea of what your IdP is returnin
 console.info(JSON.stringify(jwt, null, 2));
 ----
 
+=== Workarounds
+
+If the JWT from the OIDC identity provider does not come back with a claim called `email`, you can add your own. If the Userinfo response available to you in the lambda has a unique user, you can build a fake email address from it. 
```

Comment:
> If the JWT from the OIDC identity provider does not come back with a claim called `email`, you can add your own.

Or whatever value is resolved by the `oauth2.emailClaim` configuration parameter. This is `email` by default.  (looks like this is missing from the API at the moment). 

Fixed doc : https://github.com/FusionAuth/fusionauth-site/commit/a0ab699a5b86ddda7fdeedb5735f80251fe75d6f